app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,43 @@
 class CardsController < ApplicationController
+  before_action :set_card, only: [:show, :edit, :update, :destroy]
+
+  def show
+  end
+
   def index
     @cards = Card.all
   end
+
+  def new
+    @card = Card.new
+  end
+
+  def create
+    @card = Card.new(card_params)
+    if @card.save
+      redirect_to @card, notice: 'Book was successfully created.' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.